apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/style.scss

```diff
@@ -52,26 +55,15 @@
 			.block-editor-block-contextual-toolbar {
 				display: none;
 			}
-		}
 
-		&.block-editor-block-list__block {
-			// Need to get super specific to override the core css selectors:
-			&,
-			&.has-child-selected,
-			&.is-navigate-mode,
-			&.is-hovered {
-				> .block-editor-block-list__block-edit {
-					&::before {
-						transition: none;
-						border: none;
-						outline: none;
-						box-shadow: none;
-					}
-					> .block-editor-block-list__breadcrumb {
-						display: none;
-					}
-				}
+			& > .editor-block-list__block-edit.block-editor-block-list__block-edit::before {
+				background-color: rgba( #000000, 0.5 );
+				z-index: 1;
```

Comment:
Thanks for the low z-index number 🙇‍♀ 

For folks 👀 I did make this extension if y'all had more complex cases to debug.
https://github.com/gwwar/z-context